shared/util/phone-numbers/index.tsx

```diff
@@ -116,19 +117,29 @@ export const validateNumber = (rawNumber: string, region?: string | null) => {
 }
 
 export const formatPhoneNumber = (rawNumber: string) => {
-  const number = phoneUtil.parse(rawNumber, '')
-  return `+${number.getCountryCode()} ${phoneUtil.format(number, PNF.NATIONAL)}`
+  const phoneNumber = phoneUtil.parse(rawNumber, '')
+  return `+${phoneNumber.getCountryCode()} ${phoneUtil.format(phoneNumber, PNF.NATIONAL)}`
+}
+
+export const formatAnyPhoneNumbers = (rawText: string) => {
+  const found = rawText.match(/(\+)?(\d)+/)
+  const rawNumber = found ? found[0] : ''
+  const validatedNumber = validateNumber(rawNumber)
```

Comment:
should it also early exit here if a phone number was not found?

Reply:
It does early exit...note the `!phoneNumber` in the if statement below.